components/ReviewingAuthor.tsx

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import Image from "next/image";
+import Link from "next/link";
+import { useState } from "react";
+const ReviewingAuthor = ({ name, avatar, description }) => {
+  const [showMore, setShowMore] = useState(false);
+  const toggleShowMore = () => {
+    setShowMore(!showMore);
+  };
+
+  const FormatDescription = (description) => {
+    const des = description.split(". ");
+    const len = des.length;
+    return {
+      newAuthorDescription: des,
+      length: len,
+    };
+  };
+  const { newAuthorDescription, length } = FormatDescription(description);
+
+  return (
+    <>
+      <div className="max-w-9xl mx-auto bg-slate-000 shadow-md rounded-lg overflow-hidden flex flex-col sm:flex-row md:flex-row lg:flex-row">
+        <div className="w-3/5 self-center sm:w-1/4 p-8 flex justify-center items-center">
+          {avatar !== "n/a" && (
```

Comment:
why you are not using a ternary operator here? Is there any specific reason, if not use ternary operator so that there will be no code repetition